validation/resource-config_test.go

```diff
@@ -0,0 +1,93 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func TestResourceConfig(t *testing.T) {
+	isExcluded := map[uint64]bool{
+		888:          true,
+		997:          true,
+		11155421:     true,
+		11763071:     true,
+		129831238013: true,
+	}
+
+	checkResourceConfig := func(t *testing.T, chain *ChainConfig) {
+		rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+
+		require.NotEmpty(t, rpcEndpoint)
+
+		client, err := ethclient.Dial(rpcEndpoint)
+		require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+		contractAddress, err := Addresses[chain.ChainID].AddressFor("SystemConfigProxy")
+		require.NoError(t, err)
+
+		uint128Max, ok := new(big.Int).SetString("ffffffffffffffffffffffffffffffff", 16)
+		if !ok {
+			panic("cannot construct uint128Max")
+		}
+		desiredResourceConfig := bindings.ResourceMeteringResourceConfig{
+			MaxResourceLimit:            20000000,
+			ElasticityMultiplier:        10,
+			BaseFeeMaxChangeDenominator: 8,
+			MinimumBaseFee:              1000000000,
+			SystemTxMaxGas:              1000000,
+			MaximumBaseFee:              uint128Max,
+		} // from OP Mainnet
```

Comment:
This "ground truth" is:
* not resolved _per superchain_ (since all nonexcluded chains currently adhere anyway)
* not exported from `superchain`

Notable differences to the versioning information in `semver.yaml`. Open to suggestions to bring greater harmony.

Reply:
Do you mean the `superchain` module? I agree that makes sense.

Reply:
Yep! We should put it in whichever module will be imported by other projects.

Reply:
[38af987](https://github.com/ethereum-optimism/superchain-registry/pull/76/commits/38af98744883297c79776e15f517ed7489acb8fe)